exercises/pascals-triangle/pascals_triangle_test.go

```diff
@@ -38,15 +41,15 @@ func TestTestVersion(t *testing.T) {
 }
```

Comment:
Could use

`var testSize = len(triangleTestCases)`

instead of the const testSize above.